app.controller.js

```diff
@@ -0,0 +1,15 @@
+const {fetchTopics} = require('./app.model')
+const endpoints = require('./endpoints.json')
+
+exports.getTopics = (req, res, next)=>{
+    fetchTopics().then((topics)=>{
+        if (!topics[0]) {
+            res.status(200).send({message: 'The topics table is empty', data: topics})
+        }
+        res.status(200).send({topics})
+    })
```

Comment:
We can send back rows[0] from the model to remove a layer of complexity from the controller.

Reply:
In this I am returning all from the model. I am then only checking here in case we have an empty array otherwise I will want al the rows...